testutil/src/main/java/org/spine3/test/Tests.java

```diff
@@ -64,12 +67,24 @@ private Tests() {
      *     ...
      *     {@literal @}Test
      *     public void have_private_utility_ctor() {
-     *         assertTrue(hasPrivateParameterlessCtor(MyUtility.class));
+     *         assertHasPrivateParameterlessCtor(MyUtility.class));
      *     }
      * </pre>
-     * @return true if the class has private parameter-less constructor
      */
-    public static boolean hasPrivateParameterlessCtor(Class<?> targetClass) {
+    public static void assertHasPrivateParameterlessCtor(Class<?> targetClass) {
```

Comment:
For methods like this we have `Verify` class. Let's move `assertHasPrivateParameterlessCtor` there.